kernel/src/vm/vmar/mod.rs

```diff
@@ -1292,6 +1299,12 @@ pub fn get_intersected_range(range1: &Range<usize>, range2: &Range<usize>) -> Ra
     range1.start.max(range2.start)..range1.end.min(range2.end)
 }
 
+/// Returns whether a [`VmMapping`] described by the given flags is shared and
+/// potentially writable.
+fn is_shared_maywrite(is_shared: bool, perms: VmPerms) -> bool {
+    is_shared && perms.contains(VmPerms::MAY_WRITE)
```

Comment:
Actually I cannot see why using this method is more clearer than directly writing `is_shared && perms.contains(VmPerms::MAY_WRITE)`? This method is used only once?